Jenkinstest

```diff
@@ -0,0 +1,42 @@
+pipeline {
```

Comment:
Is this file a leftover?

Reply:
If this file is added intentionally, it should be added to CODEOWNERS file, as this is the only change in this PR that is pulling in the requirement for Janitor's review.

If this file is here accidentally we may need to add it to .gitignore to avoid it popping up later.

Reply:
Sorry, I was testing this in my local Jenkins and I did not remove it. My fault!